packages/example-getting-started/docs/7-controller.md

```diff
@@ -19,6 +19,78 @@ In addition to creating the handler functions themselves, we'll also be adding
 decorators that setup the routing as well as the expected parameters of
 incoming requests.
 
+First, we need to define our basic controller class as well as plug in our
+repository, which we'll use to perform our operations against the datasource.
+
+#### src/controllers/todo.controller.ts
+```ts
+import {repository} from '@loopback/repository';
+import {TodoRepository} from '../repositories/index';
+
+export class TodoController {
+  constructor(
+    @repository(TodoRepository.name) protected todoRepo: TodoRepository,
+  ) {}
+}
+```
+
+The `@repository` decorator will retrieve and inject an instance of the
+`TodoRepository` whenever an inbound request is being handled. The lifecycle
+of your controller object is typically a per-request lifecycle, which means that
+a new controller instance is created for each request. As a result, we want to
+inject our `TodoRepository` since the creation of these instances is more
+complex and expensive than making new controller instances.
+
+>**NOTE**: You can customize the lifecycle of *all* bindings in LoopBack 4!
+>Controllers can easily be made to use singleton lifecycles to minimize startup
+>costs. For more information, see the
+>[Dependency injection](http://loopback.io/doc/en/lb4/Dependency-injection.html)
+>section of our docs.
+
+Now that we have the repository wireup, let's create our first handler function.
+
+#### src/controllers/todo.controller.ts
+```ts
+import {post, param} from '@loopback/openapi-v2';
+import {HttpErrors} from '@loopback/rest';
+import {TodoSchema, Todo} from '../models';
+import {repository} from '@loopback/repository';
+import {TodoRepository} from '../repositories/index';
+
+export class TodoController {
+  constructor(
+    @repository(TodoRepository.name) protected todoRepo: TodoRepository,
+  ) {}
+  @post('/todo')
+  @param.body('todo', TodoSchema)
+  async createTodo(todo: Todo) {
+    if (!todo.title) {
+      return Promise.reject(new HttpErrors.BadRequest('title is required'));
+    }
+    return await this.todoRepo.create(todo);
+  }
+}
+```
+
+In this example, we're using two new decorators to provide LoopBack with
+metadata about the route, verb and the format of the incoming request body:
+
+- `@post('/todo')` creates metadata for LoopBack's [RestServer]() so that it can
+redirect requests to this function when the path and verb match.
+- `@param.body('todo', TodoSchema)` associates the OpenAPI schema for a Todo
+with the body of the request so that LoopBack can validate the format of an
+incoming request (**Note**: As of this writing, this is not yet functional).
```

Comment:
Could you clarify that it's VALIDATION that's not functional? One could mistakenly think that the decorator itself might not be functional without context.